frontend/app/routes/grant.$type.tsx

```diff
@@ -1,11 +1,14 @@
-import { LoaderFunctionArgs } from '@remix-run/node'
+import { LoaderFunctionArgs } from '@remix-run/cloudflare'
```

Comment:
```suggestion
import type { LoaderFunctionArgs } from '@remix-run/cloudflare'
```

(Should enable that TS feature that errors on such usage... types not used as values)